moped-editor/src/views/projects/projectView/ProjectNameEditable.js

```diff
@@ -114,6 +114,7 @@ const ProjectNameEditable = props => {
         if (props?.setIsEditing) props.setIsEditing(false);
         setIsEditing(false);
         setTimeout(() => setSnackbarState(DEFAULT_SNACKBAR_STATE), 3000);
```

Comment:
can this be avoided by using the `autoHideDuration` prop on the snackbar component?
https://mui.com/components/snackbars/

Reply:
Yes! Thanks for pointing this out and asking me to revisit that documentation. I had read it, seen that **_property_** and misconstrued it to be something I could put in the snackbar's state object, which obviously didn't work. I have put in the correct property now, and also adjusted the time to 5000 milliseconds as @mateoclarke suggested. Thanks for this help!